modules/local/airrflow_report/airrflow_report.nf

```diff
@@ -6,8 +6,8 @@ process AIRRFLOW_REPORT {
         error "nf-core/airrflow currently does not support Conda. Please use a container profile instead."
     }
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'docker.io/immcantation/airrflow:4.4.0':
-        'docker.io/immcantation/airrflow:4.4.0' }"
+        'docker.io/immcantation/airrflow:4.0.0':
+        'docker.io/immcantation/airrflow:4.0.0' }"
```

Comment:
```suggestion
```

Reply:
That's right, I would leave it like it is for now to be consistent with the other modules, and in case that we can finally build a biocontainer from the alakazam and shazam immcantation tools, so far that wasn't possible because the old versions were in the conda-forge channel and the new versions in the bioconda channel, but I'm trying to solve this with the conda-forge developers at the moment.